python/tests/ecl_tests/test_grid_statoil_coarse.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+#  Copyright (C) 2018  Statoil ASA, Norway.
+#
+#  The file 'test_grid_statoil_1.py' is part of ERT - Ensemble based Reservoir Tool.
+#
+#  ERT is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  ERT is distributed in the hope that it will be useful, but WITHOUT ANY
+#  WARRANTY; without even the implied warranty of MERCHANTABILITY or
+#  FITNESS FOR A PARTICULAR PURPOSE.
+#
+#  See the GNU General Public License at <http://www.gnu.org/licenses/gpl.html>
+#  for more details.
+import math
+
+try:
+    from unittest2 import skipIf
+except ImportError:
+    from unittest import skipIf
+
+from cwrap import Prototype
+from cwrap import open as copen
+
+import time
+from ecl import EclDataType, EclUnitTypeEnum
+from ecl.eclfile import EclKW, EclFile, openEclFile
+from ecl.grid import EclGrid
+from ecl.util.util import DoubleVector, IntVector
+from ecl.util.test import TestAreaContext
+from tests import EclTest, statoil_test
```

Comment:
For all of the new tests the `import` block can be significantly reduced/simplified. I *think* this should be enough:

```
from ecl.util.test import TestAreaContext
from ecl.grid import EclGrid

from tests impoty EclTest, statoil_test
```